policy/lib/tekton/task.rego

```diff
@@ -56,7 +56,7 @@ _maybe_tasks(pipeline) := _tasks if {
 # handle tasks from a slsav1 attestation
 _maybe_tasks(slsav1) := _tasks if {
 	slsav1.predicate.buildDefinition
-	_tasks := {json.unmarshal(dep.content) |
+	_tasks := {json.unmarshal(base64.decode(dep.content)) |
```

Comment:
Let's do a follow up on this, but I think if this is not valid json, or valid base64, then it just crashes opa (and ec along the way).